x/p2p/router_test.go

```diff
@@ -0,0 +1,409 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package p2p
+
+import (
+	"context"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
```

Comment:
Mega nit - just to stay inline with the rest of the repo
```suggestion
	"github.com/golang/mock/gomock"

	"github.com/stretchr/testify/require"
```